pkg/logs/message/origin.go

```diff
@@ -49,18 +48,11 @@ func (o *Origin) TagsPayload() []byte {
 	if o.LogSource.Config.SourceCategory != "" {
 		tagsPayload = append(tagsPayload, []byte("[dd ddsourcecategory=\""+o.LogSource.Config.SourceCategory+"\"]")...)
 	}
-	var tags string
-	if o.LogSource.Config.Tags != "" {
-		tags = o.LogSource.Config.Tags
-	}
-	if len(o.tags) > 0 {
-		if tags != "" {
-			tags = tags + ","
-		}
-		tags = tags + strings.Join(o.tags, ",")
-	}
-	if tags != "" {
-		tagsPayload = append(tagsPayload, []byte("[dd ddtags=\""+tags+"\"]")...)
+
+	tags := append(o.LogSource.Config.Tags, o.tags...)
```

Comment:
Question: does this re create a new array or does it update Config.Tags? In which case we'd have a big issue

Reply:
We could do the opposite, append Config.Tags to o.tags, as we don't use o.tags afterwards (and already modify it un `Tags()`

Reply:
Fixed, I played it safe